awsbatch-cli/tox.ini

```diff
@@ -0,0 +1,213 @@
+[tox]
+toxworkdir=../.tox
+envlist =
+    py{36,37,38,39}-{cov,nocov}
+    code-linters
+
+# Default testenv. Used to run tests on all python versions.
+[testenv]
+passenv =
+    CI GITHUB_*
+usedevelop =
+    cov: true
+    nocov: false
+deps =
+    -rtests/requirements.txt
+    pytest-travis-fold
+    cov: codecov
+commands =
+    nocov: pytest -n auto -l -v --basetemp={envtmpdir} --html=report.html --ignore=src tests/
+    cov: python setup.py clean --all build_ext --force --inplace
+    cov: pytest -n auto -l -v --basetemp={envtmpdir} --html=report.html --cov=src tests/
+    cov: codecov -e TOXENV
```

Comment:
have you checked what happens to codecov when we start reporting the coverage for batch cli? will this conflict with the main cli?